backend/modules/cohorts/repository.py

```
@@ -0,0 +1,547 @@
+    def get_cohort_simulations(self, cohort_id: int) -> List[CohortSimulation]:
+        """Get all simulations assigned to a cohort"""
+        if not MODELS_AVAILABLE:
+            return []
+        
+        return self.db.query(CohortSimulation).options(
+            selectinload(CohortSimulation.simulation)
+        ).join(
+            Scenario, CohortSimulation.simulation_id == Scenario.id
+        ).filter(
+            CohortSimulation.cohort_id == cohort_id,
+            Scenario.is_draft == False,
```

Ruff (0.14.8)

Avoid equality comparisons to `False`; use `not Scenario.is_draft:` for false checks

Replace with `not Scenario.is_draft`

(E712)